varnishgather

```diff
@@ -102,6 +102,9 @@ banner()
 run()
 {
 	incr
+
+	which "$1" >/dev/null 2>&1 || return 0
```

Comment:
Might want to use `command -v` here instead, which is (IIRC) POSIX rather than a csh-ism